OpenEdXMobile/build.gradle

```diff
@@ -426,6 +426,8 @@ android {
          */
         //TODO: Remove GradleCompatible after we upgrade to Android Studio 2.4
         disable 'GradleCompatible'
+        disable 'MissingTranslation'
```

Comment:
Won't it be better if we mark this option as a warning instead of disabling it?

Reply:
done!